code/game/objects/items/hot_potato.dm

```diff
@@ -0,0 +1,98 @@
+//CREATOR'S NOTE: DO NOT FUCKING GIVE THIS TO BOTANY!
+/obj/item/hot_potato

```

Comment:
@Buggy123

Reply:
I almost never play botany, i don't understand what constitutes a growable plant code-wise.

Reply:
you'd have to make a seed for it to be growable.